modules/fargate/fargate/main.tf

```diff
@@ -0,0 +1,360 @@
+# ------------------------------------------------------------------------------
+# AWS
+# ------------------------------------------------------------------------------
+data "aws_region" "current" {}
```

Comment:
This module seems to be a duplicate of `fargate-service` (or at least `fargate-service` does not seem to be used anywhere?).  I prefer `fargate/fargate` or `fargate/service` over `fargate/fargate-service`, but I'm still wondering if we cannot adapt `modules/service` so that it can be used for fargate also?

Reply:
My basic concern would be compatibility, the service module does not contain `aws_lb_listener_rule` resource. Otherwise, it could be merged.